### ISSUE TITLE
Suggestion - Add icon of My Favorite record on Listview rows in front of edit icon

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#2306]
#### Issue

Now you can choose  the favorite star icon only on Detailview of record

![favor](https://cloud.githubusercontent.com/assets/5773777/18787970/5dfa9526-81a5-11e6-98fe-aea723f0bd70.png)

I suggest to be in front of edit icon the star icon, and you save time (you must not go into detaiview of record) if you want mark the record as favorite
#### Your Environment

- SuiteCRM Version used: 7.7.4 Official DEMO of SuiteCRM

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Suggestion - Add icon of My Favorite record on Listview rows in front of edit icon](https://trello.com/c/jOGXmg8Q/71-suggestion-add-icon-of-my-favorite-record-on-listview-rows-in-front-of-edit-icon)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
